clients/client-rds-data/README.md

```diff
@@ -90,7 +90,7 @@ rDSData.send(batchExecuteStatementCommand, (err, data) => {
 The SDK can also send requests using the simplified callback style from version 2 of the SDK.
```

Comment:
nit: remove this section, as we don't plan to provide simplified callback style?